backend/app/notify/email_test.go

```diff
@@ -0,0 +1,90 @@
+package notify
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/umputun/remark/backend/app/store"
+)
+
```

Comment:
see https://github.com/go-pkgz/auth/blob/master/provider/sender/email_test.go I think those tests will provide better coverage, especially this one https://github.com/go-pkgz/auth/blob/master/provider/sender/email_test.go#L68 doing end-to-end, except the actual smtp ops

Reply:
I will do that after `Email.Send` will be written. Please don't review code until it's done: the main logic is yet absent.